docs/content/02-tutorials/07-custom-domain-name.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "g. Custom Domain Name"
+weight: 27
+---
+
+*Custom domain names* are simpler and more intuitive URLs that you can provide to your API users.
+
+When deploying PCM, you will be able to invoke APIs using the default base URL, which is composed of an ID generated by API gateway, the region and the stage name. For example:
```

Comment:
It's a bit confusing to call this a API as it's actually just the interface the user connects too. Let's rewrite this as:

> When connecting to PCM, the default domain is generated by API Gateway and includes the region and the stage name. For example: